README.rst

```diff
@@ -109,7 +109,9 @@ NI-DMM Python API Status
 +-------------------------------+----------------------+
 | NI-DMM (nidmm)                |                      |
 +===============================+======================+
-| Driver Version Tested Against | 19.1.0d11            |
+| Driver Version Tested Against | 18.1.0               |
++-------------------------------+----------------------+
+| Driver API Version            | 19.1.0d11            |
```

Comment:
This is internal to NI... but ok

Reply:
Yes, but the same information shows up in DLL version information.